hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -1236,6 +1237,29 @@ private void deleteInternal(Container container, boolean force)
               DELETE_ON_NON_EMPTY_CONTAINER);
         }
       }
+      if (container.getContainerData() instanceof KeyValueContainerData) {
+        KeyValueContainerData keyValueContainerData =
+            (KeyValueContainerData) container.getContainerData();
+        HddsVolume hddsVolume = keyValueContainerData.getVolume();
+
+        // Rename container location
+        boolean success = KeyValueContainerUtil.ContainerDeleteDirectory
+            .moveToTmpDeleteDirectory(keyValueContainerData, hddsVolume);
+
+        if (success) {
```

Comment:
This block (success) can be restructured to only be used and output only if debug logging is enabled.  Something like:

```
        if (!success) {
          LOG.error("Failed to move container under " +
              hddsVolume.getDeleteServiceDirPath());
          throw new StorageContainerException("Moving container failed",
              CONTAINER_INTERNAL_ERROR);
        }
        if (LOG.isDebugEnabled()) {
          String containerPath = keyValueContainerData
              .getContainerPath().toString();
          File containerDir = new File(containerPath);

          LOG.debug("Container {} has been successfuly moved under {}",
              containerDir.getName(), hddsVolume.getDeleteServiceDirPath());
        }
```